src/pages/en/guides/markdown-content.md

```diff
@@ -172,6 +172,10 @@ You can also pass the `--drafts` flag when running `astro build` to build draft
 
 ## Authoring Markdown
 
+:::caution[Deprecated]
+Astro no longer supports components or JSX in markdown pages by default and may be removed in a future release. In the meantime, Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will enable these features in markdown pages until you are able to migrate to [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/).
```

Comment:
```suggestion
Astro no longer supports components or JSX in Markdown pages by default and may be removed in a future release. In the meantime, Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will enable these features in Markdown pages until you are able to migrate to [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/).
```
We capitalize Markdown in the docs, so I've changed those.

Just crossed my mind: is it worth pointing out that there are two options: migrate to using MDX *or* traditional Markdown (i.e. remove your comonents/JSX)?  Might not be, but it's technically a possibility.